staging/src/k8s.io/kubectl/pkg/util/term/term_writer.go

```diff
@@ -74,6 +74,28 @@ func NewWordWrapWriter(w io.Writer, limit uint) io.Writer {
 	}
 }
 
+func GetWordWrapperLimit() uint {
+	stdout := os.Stdout
+	fd := stdout.Fd()
+	if !term.IsTerminal(fd) {
+		panic("file descriptor is not a terminal")
+	}
+	terminalSize := GetSize(fd)
+	if terminalSize == nil {
+		panic("terminal size is nil")
+	}
+	var limit uint
+	switch {
+	case terminalSize.Width >= 120:
```

Comment:
Can you please help me understand these breakpoints?

Reply:
I just followed what the [`ResponsiveWriter`](https://github.com/kubernetes/kubernetes/blob/73d51a21dbe7504a00423877e272a279a21a115f/staging/src/k8s.io/kubectl/pkg/util/term/term_writer.go#L40) intended to do. Because it's returning a writer, but to get the `limit`, I have to borrow the idea from the function

Reply:
Can we pull that out to a function?

Reply:
Yes! that would be better